dev/io.openliberty.microprofile.openapi.internal.common/resources/io/openliberty/microprofile/openapi/internal/resources/OpenAPI.nlsprops

```diff
@@ -98,3 +98,43 @@ OPENAPI_MERGE_INVALID_NAME_CWWKO1666W.useraction=Review the error text to identi
+OPEN_API_DOC_PATH_UPDATE_CWWKO1669I.useraction=No action is required.
+
+OPEN_API_UI_PATH_UPDATE_FAILED_CWWKO1670E=CWWKO1670E: The OpenAPI UI path is invalid, the default value will be used instead. Check previous warnings for cause of validation failure.
+OPEN_API_UI_PATH_UPDATE_FAILED_CWWKO1670E.explanation= The OpenAPI UI path is invalid and will use the default value instead.
+OPEN_API_UI_PATH_UPDATE_FAILED_CWWKO1670E.useraction=Check earlier error messages for causes of validation failures.
+
+OPEN_API_DOC_PATH_UPDATE_FAILED_CWWKO1671E=CWWKO1671E: The OpenAPI Document path is invalid, the default value will be used instead. Check previous warnings for cause of validation failure.
+OPEN_API_DOC_PATH_UPDATE_FAILED_CWWKO1671E.explanation= The OpenAPI Document path is invalid and will use the default value instead.
+OPEN_API_DOC_PATH_UPDATE_FAILED_CWWKO1671E.useraction=Check earlier error messages for causes of validation failures.
+
+OPEN_API_UI_PATH_CONFLICT_CWWKO1672E=CWWKO1672E: The OpenAPI UI path cannot be set to {0}. This path is used to retrieve the OpenAPI documents.
+OPEN_API_UI_PATH_CONFLICT_CWWKO1672E.explanation=The OpenAPI Document and OpenAPI UI paths cannot be the same.
+OPEN_API_UI_PATH_CONFLICT_CWWKO1672E.useraction=Specify a unique path for the OpenAPI UI.
+
+OPEN_API_UI_PATH_POTENTIAL_CONFLICT_CWWKO1673W=CWWKO1673W: The OpenAPI UI path was set to the same path as the {0} feature, which might prevent startup of itself or the other feature.
+OPEN_API_UI_PATH_POTENTIAL_CONFLICT_CWWKO1673W.explanation=An OpenAPI UI path was set to the same path as another feature, which can prevent startup or access.
+OPEN_API_UI_PATH_POTENTIAL_CONFLICT_CWWKO1673W.useraction=Specify a different path for the OpenAPI path.
+
+OPEN_API_DOC_PATH_POTENTIAL_CONFLICT_CWWKO1674W=CWWKO1674W: The OpenAPI Document path was set to the same path as the {0} feature, which might prevent startup of itself or the other feature.
+OPEN_API_DOC_PATH_POTENTIAL_CONFLICT_CWWKO1674W.explanation=An OpenAPI Document path was set to the same path as another feature, which can prevent startup or access.
+OPEN_API_DOC_PATH_POTENTIAL_CONFLICT_CWWKO1674W.useraction=Specify a different path for the OpenAPI path.
+
+OPEN_API_UI_PATH_INVALID_CWWKO1675E=CWWKO1675E: The OpenAPI UI path is invalid or contains invalid characters. The URL must be constructed using Unicode alphanumeric characters A-Za-z0-9, the underscore (_), dash (-), forward slash (/) and period (.).
+OPEN_API_UI_PATH_INVALID_CWWKO1675E.explanation=The OpenAPI UI Path cannot be set to a value that contains invalid characters.
+OPEN_API_UI_PATH_INVALID_CWWKO1675E.useraction=Ensure the OpenAPI Path contains only A-Za-z0-9, _, -, ., /.
+
+OPEN_API_DOC_PATH_INVALID_CWWKO1676E=CWWKO1676: The OpenAPI Document path is invalid or contains invalid characters. The URL must be constructed using Unicode alphanumeric characters A-Za-z0-9, the underscore (_), dash (-), forward slash (/) and period (.).
```

Comment:
L2 suggested edit to remove the word "the" prior to "underscore", to maintain parallelism in the list of valid characters:
`OPEN_API_DOC_PATH_INVALID_CWWKO1676E=CWWKO1676: The OpenAPI Document path is invalid or contains invalid characters. The URL must be constructed using Unicode alphanumeric characters A-Za-z0-9, underscore (_), dash (-), forward slash (/), or period (.).`